docs/react/faq.en-US.md

```diff
@@ -268,13 +268,48 @@ The above problem occurs if `strictNullChecks` is set to `true`, If you can dete
 
 ## The antd component reported an error when using the App Router of Next.js
 
-If you are using the App Router of Next.js, when you use the sub-components provided by some antd components, such as `Select.Option `, `Form.Item`, etc., you may get the following error:
+If you are using the App Router of Next.js, when you use the sub-components provided by some antd components, such as `Select.Option `, `Form.Item`, `Typography.Title`, etc., you may get the following error:
 
 ```bash
 Error: Cannot access .Option on the server. You cannot dot into a client module from a server component. You can only pass the imported name through.
 ```
 
-At present, this problem is waiting for Next.js to give an official solution, before this, if you use sub-components in your page, you can try to add the following client tag at the top of the page to solve this problem:
+Currently, this problem is [waiting for Next.js to give an official solution](https://github.com/vercel/next.js/issues/51593). There are two workarounds as of now if you need to use sub-components in your page with the App Router:
```

Comment:
Please help to add Chinese documents changes also.

Reply:
Unfortunately, I can't speak Chinese, and the best I could do it to use Google Translate, DeepL machine translation.

Is that acceptable?

Reply:
Yes, and we can suggest you after you add that changes.